mobile/lib/resources/ra_links.dart

```diff
@@ -1,9 +1,7 @@
 abstract class RaRadio {
-  static const int playerPort = 8443;
-
-  static const String baseUrl = 'listen.radioaktywne.pl:$playerPort';
```

Comment:
I'd leave this one in the previous form (with updated port of course, which is 443, at least at the moment). My reasoning is that RA are going to have ability to stream on both 443 and 8443 port, which may be used for the transition between the old and new IceCast streaming server. That's why I see this as useful still.